docs/server/v20/server/introduction/clients.md

```diff
@@ -25,29 +32,36 @@ EventStoreDB offers a low-level protocol in the form of an asynchronous TCP prot
 We plan to phase out the TCP protocol in the later versions. Please consider migrating your applications that use the TCP protocol and refactor them to use gRPC instead.
 :::
 
+Find out more about configuring the TCO protocol on the [TCP configuration](../networking/tcp.md) page.
```

Comment:
TCP instead of TCP

Reply:
Good catch! Fixed in https://github.com/EventStore/documentation/pull/238/commits/c8595d8148de279abe1558fb37f8e3f4de0e97a5.